examples/ancestry/run_ancestry_example.py

```
@@ -0,0 +1,495 @@
+    accuracy = result.get_accuracy()
+    if accuracy:
+        print(f"\n  Cross-validation accuracy: {accuracy:.2%}")
+
+    # Variance explained
+    var_explained = result.variance_explained()
+    print(f"\n  Variance explained by PC1: {var_explained[0]:.2%}")
+    print(f"  Variance explained by PC1-5: {sum(var_explained[:5]):.2%}")
+
+    # Pipeline statistics
+    stats = result.pipeline_stats
+    print(f"\n  Pipeline statistics:")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)